File: src/kyu_8/is_it_a_number/js/isDigitTest.js
const Test = require('@codewars/test-compat');
const isDigit = require("./isDigit");

describe("Solution", function(){
    it("basic tests", function(){
        var input = "s2324";
        Test.assertEquals(isDigit(input), false);

        input = "-234.4";
        Test.assertEquals(isDigit(input), true);
    });
});

File: src/kyu_8/regexp_basics_is_it_a_digit/python/is_digit_test.py
import codewars_test as test
from solution import is_digit

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(is_digit(""), False)
        test.assert_equals(is_digit("7"), True)
        test.assert_equals(is_digit(" "), False)
        test.assert_equals(is_digit("a"), False)
        test.assert_equals(is_digit("a5"), False)